backend/src/personnel/dto/bcws/create-bcws-personnel.dto.ts

```diff
@@ -0,0 +1,199 @@
+import { ApiProperty } from '@nestjs/swagger';
+import {
+  IsEmail,
+  IsEnum,
+  IsOptional,
+  IsString,
+  Length,
+  ValidateIf,
+} from 'class-validator';
+import { CreateBcwsCertificationsDTO } from './create-bcws-personnel-certifications.dto';
+import { CreateBcwsPersonnelLanguagesDTO } from './create-bcws-personnel-languages.dto';
+import { CreateBcwsPersonnelRolesDTO } from './create-bcws-personnel-roles.dto';
+import { CreateBcwsPersonnelToolsDTO } from './create-bcws-personnel-tools.dto';
+import { BcwsRole, FireCentre, OFA } from '../../../common/enums/bcws';
+import { Status } from '../../../common/enums/status.enum';
+
+class BcwsLocationDTO {
+  @IsOptional()
+  id?: number;
+
+  @IsString()
+  locationName?: string;
+
+  @ApiProperty({
+    description: 'Ministry personnel works in',
+    enum: FireCentre,
+    example: FireCentre.CARIBOO,
+  })
+  @IsEnum(FireCentre)
+  fireCentre?: FireCentre;
+}
+
+export class CreatePersonnelBcwsDTO {
```

Comment:
Should this extend PersonnelDTO?

Reply:
no, we'll want to create a single person, and then create bcws/emcr data (otherwise if they both extend person and we call create on each of them for someone applying to both programs we'll have two personnel entities)

Reply:
I follow for now. May want to change in the future.